eventhubs/data-plane/azure-eventhubs-eph/pom.xml

```diff
@@ -1,19 +1,23 @@
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>azure-eventhubs-eph</artifactId>
+
+    <name>Microsoft Azure SDK for Event Hubs Eph</name>
+    <description>libraries and extensions built on Microsoft Azure Event Hubs</description>
```

Comment:
This seems very generic for a description. I know this is what was there previously, but can you be any more specific (and use upper case letters at the start of the sentence)?